deploy/.env.example

```
@@ -0,0 +1,21 @@
+# =============================================================================
+# AIOX Squads — Variáveis de Ambiente (Produção)
+# Copie para .env e preencha antes de iniciar: cp .env.example .env
+# NUNCA commite o arquivo .env no repositório!
+# =============================================================================
+
+# ── Anthropic API ─────────────────────────────────────────────────────────────
+ANTHROPIC_API_KEY=sk-ant-COLOQUE_SUA_CHAVE_AQUI
+
+# ── Aplicação ─────────────────────────────────────────────────────────────────
+APP_DOMAIN=aiox.seudominio.com.br       # Seu domínio (sem https://)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)